pkg/infrastructure/services/deployment.go

```diff
@@ -74,3 +79,23 @@ func createRequiredDeployment(service v1alpha1.KogitoService, resolvedImage stri
 
 	return deployment
 }
+
+// getServiceHTTPPort gets the service port for the given KogitoService based on httpPort CR parameter.
+// defaults to 8080
+func getServiceHTTPPort(kogitoService v1alpha1.KogitoService) int32 {
+	// port should be greater than 0
+	httpPort := kogitoService.GetSpec().GetHTTPPort()
+	if httpPort < 1 {
+		log.Debugf("HTTPPort not set, returning default http port.")
+		return framework.DefaultExposedPort
+	}
+	log.Debugf("HTTPPort is set, returning port number %i", httpPort)
+	return httpPort
+}
+
+// SetHTTPPortInEnvVar will update or add the environment variable into the given kogito service
+func SetHTTPPortInEnvVar(httpPort int32, kogitoService v1alpha1.KogitoService) {
```

Comment:
I believe this function also can be private.

Reply:
done